### ISSUE TITLE
Unable to create new todos 

### ISSUE THREAD

**Issue by MeganSelkirk** [SuperChamp234/habitica-sync#33]
This tool would be perfect if I could update my todos through obsidian. This is a feature request rather than a criticism.

**Comment by neuromaancer** [comment]
If we could create the todo directly in obsidian that would be great!

**Comment by Vyeche** [comment]
Yep, agree; bidirectional sync for todos would be a great feature.
